packages/blocks/chart/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@hashintel/block-chart",
+  "version": "0.0.0",
+  "description": "A block for plotting entities in a 2D chart",
+  "keywords": [
+    "blockprotocol",
+    "blocks"
+  ],
+  "license": "MIT",
+  "author": "Ben Werner",
+  "scripts": {
+    "build": "block-scripts build",
+    "dev": "block-scripts dev",
+    "fix:eslint": "eslint --ext .ts,.tsx --fix ./src/",
+    "lint:eslint": "eslint --ext .ts,.tsx ./src/",
+    "lint:tsc": "tsc --noEmit",
+    "serve": "block-scripts serve"
+  },
+  "dependencies": {
+    "@mui/material": "5.4.3",
+    "blockprotocol": "0.0.8",
+    "echarts": "^5.3.2",
+    "lodash.debounce": "4.0.8"
+  },
+  "devDependencies": {
+    "@types/lodash.debounce": "4.0.7",
+    "block-scripts": "0.0.6",
+    "mock-block-dock": "0.0.8",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "peerDependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "block-scripts": {
+    "devPort": 9090,
+    "servePort": 63212
```

Comment:
[optional]

```suggestion
    "port": SOME NUMBER THAT IS NOT USED ELSEWHERE
```

Reply:
Done in https://github.com/hashintel/hash/pull/507/commits/46659cbf11126f0b8712b58dbf04f2814c0c0f39